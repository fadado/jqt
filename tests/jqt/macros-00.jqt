<# define required partials #>&
<%shortcode youtube>&
<%shortcode analytics>&
<# expand partials #>&
<%youtube 7zIoLvbCCm8 420 315>
<%analytics UA-8673551-2>
