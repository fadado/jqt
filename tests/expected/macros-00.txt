<iframe src="https://www.youtube.com/embed/7zIoLvbCCm8"
        width="420"
        height="315"
        frameborder="0"></iframe>

<script type="text/javascript">
    var _gaq = _gaq || [];
    _gaq.push(["_setAccount", "UA-8673551-2"]);
    _gaq.push(["_trackPageview"]);
    (function() {
        var ga = document.createElement("script");
        ga.type = "text/javascript";
        ga.async = true;
        ga.src = ("https:" == document.location.protocol ? "https://ssl" : "http://www") + ".google-analytics.com/ga.js";
        var s = document.getElementsByTagName("script")[0];
        s.parentNode.insertBefore(ga, s);
    })();
</script>

